plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySplitManager.java

```diff
@@ -85,13 +85,13 @@ public ConnectorSplitSource getSplits(
         log.debug("getSplits(transaction=%s, session=%s, table=%s, splitSchedulingStrategy=%s)", transaction, session, table, splitSchedulingStrategy);
         BigQueryTableHandle bigQueryTableHandle = (BigQueryTableHandle) table;
 
-        TableId tableId = bigQueryTableHandle.getRemoteTableName().toTableId();
+        TableId remoteTableId = bigQueryTableHandle.getRemoteTableName().toTableId();
```

Comment:
nit: if possible we can extract variable renames to it's own commit (since they are no-op changes)

Because changes in BigQuerySplitManager and ReadSessionCreator both are renames only.

Reply:
I now think it'd be much easier to figure out the actual bugfix after extracting a commit.